.github/workflows/land-blocking.yml

```diff
@@ -7,27 +7,22 @@ on:
       - auto
 
 jobs:
-  build-and-run-cluster-test:
-    name: Build images and run cluster test
+  build-images:
```

Comment:
Not sure if you are aware about this: when the build fails here, the PR won't get detailed message like the cluster test. The PR will get link to the failed run.  I think this should be fine. But I want to call it out to your attention.